src/typeob.rs

```diff
@@ -195,8 +195,9 @@ pub(crate) unsafe fn pytype_drop<T: PyTypeInfo>(py: Python, obj: *mut ffi::PyObj
 }
 
 /// A Python object allocator that is usable as a base type for `#[pyclass]`
-pub trait PyObjectAlloc: PyTypeInfo {
+pub trait PyObjectAlloc: PyTypeInfo + Sized {
     /// Allocates a new object (usually by calling ty->tp_alloc),
+<<<<<<< HEAD
```

Comment:
Hoops looks git inserted a garbage here

Reply:
Oh thanks I failed to rebase properly. Fixing this now